src/commands/ai/text-gen.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const { flags } = require('@oclif/command');
+
+class AiTextGenCommand extends BoxCommand {
+	async run() {
+		const { flags, args } = this.parse(AiTextGenCommand);
+		let options = {};
+
+        if (flags.dialogue_history) {
+            options.dialogueHistory = flags.dialogue_history;
+        }
+        options.prompt = flags.prompt;
+        options.items = flags.items;
+        let answer = await this.client.ai.textGen({
+            prompt: options.prompt,
+            items: options.items,
+            dialogue_history: options.dialogueHistory,
+        });
+
+        await this.output(answer);
+	}
+}
+
+AiTextGenCommand.description = 'Sends an AI request to supported LLMs and returns an answer specifically focused on the creation of new text.';
+AiTextGenCommand.examples = ['box ai:text-gen --dialogue_history=prompt="What is the status of this document?",answer="It is in review" --items=id=12345,type=file --prompt="What is the status of this document?"'];
```

Comment:
The example is missing the `created-at` field which is required in the request, so it needs to be added here.